irispy/iris_tools.py

```diff
@@ -347,8 +357,8 @@ def fit_iris_xput(time_obs, time_cal_coeffs, cal_coeffs):
 
     Parameters
     ----------
-    time_obs: a numpy array of floats
-    - Times of datapoints.
+    time_obs: a `numpy.array`
+        A set of observation times contained in a numpy array.
```

Comment:
Say that each element in the array must be an astropy Time object.

Reply:
But for the `fit_iris_xput` the `time_obs` is indeed a numpy array.

Reply:
but you need to say that each element in the array have to be an astropy Time object - i.e. say that the set of observation times are astropy times

Reply:
Okay, will do